x-pack/plugins/timelines/server/search_strategy/timeline/eql/helpers.test.ts

```diff
@@ -178,273 +178,7 @@ describe('Search Strategy EQL helper', () => {
-                ],
-                "_id": "qxymg3cBX5UUcOOYP3Ec",
-                "_index": ".ds-logs-endpoint.events.security-default-2021.02.05-000005",
-                "agent": Object {
-                  "id": Array [
-                    "1d15cf9e-3dc7-5b97-f586-743f7c2518b2",
-                  ],
-                  "type": Array [
-                    "endpoint",
-                  ],
-                },
-                "event": Object {
-                  "action": Array [
-                    "log_on",
-                  ],
-                  "category": Array [
-                    "authentication",
-                    "session",
-                  ],
-                  "created": Array [
-                    "2021-02-08T21:50:28.3377142Z",
-                  ],
-                  "dataset": Array [
-                    "endpoint.events.security",
-                  ],
-                  "id": Array [
-                    "LzzWB9jjGmCwGMvk++++FG/P",
-                  ],
-                  "kind": Array [
-                    "event",
-                  ],
-                  "module": Array [
-                    "endpoint",
-                  ],
-                  "outcome": Array [
-                    "success",
-                  ],
-                  "type": Array [
-                    "start",
-                  ],
-                },
-                "host": Object {
-                  "id": Array [
-                    "d8ad572e-d224-4044-a57d-f5a84c0dfe5d",
-                  ],
-                  "ip": Array [
-                    "10.128.0.57",
-                    "fe80::9ced:8f1c:880b:3e1f",
-                    "127.0.0.1",
-                    "::1",
-                  ],
-                  "name": Array [
-                    "win2019-endpoint-mr-pedro",
-                  ],
-                  "os": Object {
-                    "family": Array [
-                      "windows",
-                    ],
-                    "name": Array [
-                      "Windows",
-                    ],
-                  },
-                },
-                "message": Array [
-                  "Endpoint security event",
-                ],
-                "process": Object {
-                  "entity_id": Array [
-                    "MWQxNWNmOWUtM2RjNy01Yjk3LWY1ODYtNzQzZjdjMjUxOGIyLTU4MC0xMzI1NTA3ODY2Ny45MTg5Njc1MDA=",
-                  ],
-                  "executable": Array [
-                    "C:\\\\Windows\\\\System32\\\\lsass.exe",
-                  ],
-                },
-                "timestamp": "2021-02-08T21:50:28.3377142Z",
-                "user": Object {
-                  "domain": Array [
-                    "NT AUTHORITY",
-                  ],
-                  "name": Array [
-                    "SYSTEM",
-                  ],
-                },
-              },
-            },
-          },
-        ]
-      `);
+      expect(result.edges).toMatchInlineSnapshot(`Array []`);
```

Comment:
I think these snapshot changes are due to the accompanying mocks (in `__mocks__`) being outdated.

Reply:
yep exactly if you remove the body key from https://github.com/elastic/kibana/blob/main/x-pack/plugins/timelines/server/search_strategy/timeline/eql/__mocks__/index.ts#L13 and https://github.com/elastic/kibana/blob/main/x-pack/plugins/timelines/server/search_strategy/timeline/eql/__mocks__/index.ts#L580 these snapshots will pass as before